docs/constrained_spherical_deconvolution/single_tissue_csd.rst

```diff
@@ -1,14 +1,14 @@
-.. _csd:
+.. single_tissue_csd:
 
-constrained spherical deconvolution
-===================================
+Single-tissue constrained spherical deconvolution
```

Comment:
Not really comfortable with changing 'CSD' to 'single-tissue CSD'. You can happily perform single-tissue CSD with the MSMT algorithm. In fact, there's [instructions on how to do just that in the FBA section](http://mrtrix.readthedocs.io/en/dev/fixel_based_analysis/st_fibre_density_cross-section.html?highlight=msmt_csd#fibre-orientation-distribution-estimation-spherical-deconvolution)... We could at a push call it 'single-shell CSD' - I'd be more comfortable with that, but it doesn't get around the fact that it can be done using MSMT-CSD... 

I think we should stick with the simple 'CSD' acronym, since that's how it's been called since it was originally proposed. I don't think leaving it as-is is going to cause any confusion particularly - in fact, I reckon modifying the existing label is more likely to lead to confusion (e.g. how does 'single-shell CSD' differ from regular 'CSD'?).

Reply:
Also, and more to the point: you can do single-tissue, _multi-shell_ CSD with MSMT-CSD - you definitely can't with regular CSD. That's why I'm (slightly) more comfortable with 'single-shell CSD' than 'single-tissue CSD' - but I'd still rather stick to 'CSD'.

Reply:
So ok, in the interest of time, I'm ok with "CSD" (though not so much / at all with "single-shell CSD").  I'll fix it again.  Not intending to change that either in the long run or anything, but it came from a different point of view; the gist being that the name emphasises the capabilities rather than the requirements.  Most models' names refer (mainly) to what compartments / degrees of freedom / etc... they model; i.e. refer to their parameters.  What data is needed to fit that, is a bit more secondary to that (though perfectly important as well; I agree there).  Well, my opinion is far longer and more nuanced than that; but we'll clearly find a more common ground in sticking with CSD, which is indeed also simply the original name / reference it goes by. :+1: